src/parser.pegjs

```diff
@@ -48,3 +52,64 @@ digit
 
 hex_digit
 	= [0-9a-f]i
+
+// RSQL "quoted" values extesion
+// inspired by https://github.com/pegjs/pegjs/blob/master/examples/javascript.pegjs#L271
+
+string "string"
```

Comment:
Possibly the name `quoted_string` is more appropriate instead of just `string`? I.e. possibly better `quoted_string "string"` here?